ddsketch/ddsketch.py

```diff
@@ -217,9 +206,36 @@ def copy(self, sketch):
 
 class DDSketch(BaseDDSketch):
     """The default implementation of BaseDDSketch, with optimized memory usage at
-    the cost of lower ingestion speed, using a limited number of bins. When the
-    maximum number of bins is reached, bins with lowest indices are collapsed,
-    which causes the relative accuracy to be lost on lowest quantiles. For the
+    the cost of lower ingestion speed, using an unlimited number of bins. The
+    number of bins will not exceed a reasonable number unless the data is
+    distributed with tails heavier than any  subexponential.
+    (cf. http://www.vldb.org/pvldb/vol12/p2195-masson.pdf)
+    """
+
+    def __init__(self, relative_accuracy=None):
+
+        # Make sure the parameters are valid
+        if relative_accuracy is None or (
+            relative_accuracy <= 0 or relative_accuracy >= 1
```

Comment:
In both [Go](https://github.com/DataDog/sketches-go/blob/master/ddsketch/mapping/logarithmic_mapping.go#L25) and [Java](https://github.com/DataDog/sketches-java/blob/master/src/main/java/com/datadoghq/sketch/ddsketch/mapping/LogLikeIndexMapping.java#L25), this is an error. Should we do the same in Python instead of silently changing the value?